src/audiovideocontroller/DefaultAudioVideoController.ts

```diff
@@ -952,6 +952,7 @@ export default class DefaultAudioVideoController
     if (added.length !== 0 || removed.length !== 0) {
       return false;
     }
+    context.videoStreamIndex.subscribeFrameSent();
```

Comment:
[For understanding]: So the above `context.signalingClient.remoteVideoUpdate` means sending of a subscribe frame?

Reply:
Its not actually sending a subscribe, I will update the comment in docstring and above usage. Alternatively I could create an alias function `cacheCurrentIndex` but maybe will save for a new major rev :)

Reply:
Cool, thanks for explaining and elaborating the docstring.